javascripts/discourse/connectors/above-latest-topic-list-item/author-avatar-connector.gjs

```diff
@@ -0,0 +1,11 @@
+import Component from "@ember/component";
+import { classNames } from "@ember-decorators/component";
+import CustomCategoriesLatestAvatar from "../../components/custom-categories-latest-avatar";
+
+@classNames("above-latest-topic-list-item-outlet", "author-avatar-connector")
+export default class AuthorAvatarConnector extends Component {
+  <template>
+    {{! has-modern-replacement }}
```

Comment:
Not sure why this is here... this comment was only supposed to be used in raw-hbs files 🤷 
```suggestion
```